ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/FilterParameter.java

```diff
@@ -33,34 +34,26 @@
   private final BlockParameter fromBlock;
   private final BlockParameter toBlock;
   private final List<Address> addresses;
-  private final TopicsParameter topics;
+  private final List<List<LogTopic>> topics;
   private final Hash blockhash;
 
   @JsonCreator
   public FilterParameter(
       @JsonProperty("fromBlock") final String fromBlock,
       @JsonProperty("toBlock") final String toBlock,
       @JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY) @JsonProperty("address")
-          final List<String> address,
+          final List<Address> address,
       @JsonDeserialize(using = TopicsDeserializer.class) @JsonProperty("topics")
-          final TopicsParameter topics,
+          final List<List<LogTopic>> topics,
       @JsonProperty("blockhash") final String blockhash) {
     this.fromBlock =
         fromBlock != null ? new BlockParameter(fromBlock) : new BlockParameter("latest");
     this.toBlock = toBlock != null ? new BlockParameter(toBlock) : new BlockParameter("latest");
-    this.addresses = address != null ? renderAddress(address) : Collections.emptyList();
-    this.topics = topics != null ? topics : new TopicsParameter(Collections.emptyList());
+    this.addresses = Optional.ofNullable(address).orElse(emptyList());
```

Comment:
```suggestion
    this.addresses = Optional.ofNullable(address).orElseGet(Collections::emptyList);
```
is the magic sauce you want using `Optional`.  It will only create the empty list if it's actually required.

Although in this case since we're not returning an Optional, I'd probably lean towards the ternary for style as well. :)

Reply:
Yeah, I knew the `orElseGet` was an option but I just switched back to ternary for the reason you mentioned